runtime/runtime_controller.h

```diff
@@ -22,13 +22,89 @@
+  /// @param[in]  isolate_snapshot            The isolate snapshot used to start
+  ///                                         the root isolate managed by this
+  ///                                         runtime controller. The isolate
+  ///                                         must be transitioned into the
+  ///                                         running phase manually by the
+  ///                                         caller.
+  /// @param[in]  task_runners                The task runners used by the shell
+  ///                                         hosting this runtime controller.
+  ///                                         This may be used by the isolate to
+  ///                                         scheduled asynchronous texture
+  ///                                         uploads or post tasks to the
+  ///                                         platform task runner.
+  /// @param[in]  snapshot_delegate           The snapshot delegate used by the
+  ///                                         isolate to gather raster snapshots
+  ///                                         of Flutter view hierarchies.
+  /// @param[in]  io_manager                  The IO manager used by the isolate
+  ///                                         for asynchronous texture uploads.
+  /// @param[in]  unref_queue                 The unref queue used by the
+  ///                                         isolate to collect resources that
+  ///                                         may reference resources on the
+  ///                                         GPU.
+  /// @param[in]  image_decoder               The image decoder
+  /// @param[in]  advisory_script_uri         The advisory script URI (only used
+  ///                                         for debugging). This does not
+  ///                                         affect the code being run in the
+  ///                                         isolate in any way.
+  /// @param[in]  advisory_script_entrypoint  The advisory script entrypoint
+  ///                                         (only used for debugging). This
+  ///                                         does not affect the code being run
+  ///                                         in the isolate in any way. The
+  ///                                         isolate must be transitioned to
+  ///                                         the running state explicitly by
+  ///                                         the caller.
+  /// @param[in]  idle_notification_callback  The idle notification callback.
+  ///                                         This allows callers to run native
+  ///                                         code in isolate scope when the VM
+  ///                                         is about to be notified that the
+  ///                                         engine is going to be idle.
+  /// @param[in]  window_data                 The window data (if exists).
```

Comment:
What's window data? :)

Reply:
Maybe even just a link to the WindowData struct would do.

Reply:
Doxygen seem to be inferring a link the struct already https://engine.chinmaygarde.com/classflutter_1_1_runtime_controller.html#af7c1055c85ff7300fdb2fd2d9576440a